applications/serial_lte_modem/src/twi/Kconfig

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+config SLM_TWI
+	bool "Access TWI interface with SLM AT command"
+	select I2C
```

Comment:
After PR#4153, must add "default y" here

Reply:
Defaut y if there is any i2c instance in device tree.

config SLM_TWI
	bool "Access TWI interface with SLM AT command"
	default y
	select I2C
	depends on "$(dt_nodelabel_enabled,i2c0)" || \
		   "$(dt_nodelabel_enabled,i2c1)" || \
		   "$(dt_nodelabel_enabled,i2c2)" || \
		   "$(dt_nodelabel_enabled,i2c3)"

Reply:
default y if "$(dt_nodelabel_enabled,i2c0)" ||
"$(dt_nodelabel_enabled,i2c1)" ||
"$(dt_nodelabel_enabled,i2c2)" ||
"$(dt_nodelabel_enabled,i2c3)"